### PR TITLE
Remove unused checks

### LINKED ISSUE #669: Operator Package name no longer used and should be removed
Operator Package Name Uniqueness check uses it's own code for interacting with Pyxis. It should use the pyxis package.

### PR DESCRIPTION
**Pull request by bcrochet**
There is a check that is completely unused, and some other checks had
comments still in the engine.go. Remove the cruft.

Fixes #669

Signed-off-by: Brad P. Crochet <brad@redhat.com>

### CODE REVIEW

File: certification/engine/engine.go [hunk 1]
```diff
@@ -99,12 +99,9 @@ var (
 		viper.GetString("pyxis_api_token"),
 		viper.GetString("certification_project_id"),
 		&http.Client{Timeout: 60 * time.Second}))
-	// runnableContainerCheck  certification.Check = containerpol.NewRunnableContainerCheck(internal.NewPodmanEngine())
```
Comment (acornett21) [inline certification/engine/engine.go:102]: Did we already remove the implementations for these? Or are you thinking that will be in another PR?

**Review by komish — COMMENTED** [review]
/lgtm

**Review by komish — APPROVED** [review]

**Review by acornett21 — APPROVED** [review]
